src/core/surface/call.c

```diff
@@ -40,6 +40,7 @@
 #include "src/core/surface/byte_buffer_queue.h"
 #include "src/core/surface/channel.h"
 #include "src/core/surface/completion_queue.h"
+#include <grpc/census.h>
```

Comment:
We can't include census related code directly in this one. It creates a physical dependency and I want to be able to disable census completely without recompiling.